lib/rubycritic/platforms/darwin.rb

```diff
@@ -0,0 +1,26 @@
+require "rubycritic/platforms/base"
+
+module Rubycritic
+  module Platforms
+    class Darwin < Base
+      BROWSER_MAP = {}.tap do |hash|
+        hash.merge! :chrome => "Google Chrome", :firefox => "firefox", :safari => "safari"
```

Comment:
just to be sure, is it really `Google Chrome` instead of `google-chrome`? (I don't have a mac so, I can't assess this myself)